crates/parser/test_data/parser/inline/ok/0106_lambda_expr.rs

```diff
@@ -6,4 +6,5 @@ fn foo() {
     async || {};
     move || {};
     async move || {};
+    static || {};
```

Comment:
Can you also include `static move || {};`?

Reply:
And also `static async move`  ?

https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=48be355aed75f9e83f240e578f410c69

Reply:
You are right. I am going to had it too